sdks/python/build.gradle

```diff
@@ -37,7 +38,10 @@ task setupVirtualenv {
       args '-c', ". ${envdir}/bin/activate && pip install --upgrade tox"
     }
   }
-  outputs.files("${envdir}/bin/tox")
+  // Gradle will delete outputs whenever it thinks they are stale. Putting a
```

Comment:
Is this a warning to reader? Or is there more work that can be done?

Reply:
I intended it as documentation about why we're using outputs.dirs vs outputs.files.